Libraries/Components/Pressable/useAndroidRippleForView.js

```diff
@@ -39,56 +46,60 @@ export default function useAndroidRippleForView(
+  if (!isSupported || !rippleConfig) {
+    return null;
+  }
   return useMemo(() => {
-    if (
-      Platform.OS === 'android' &&
-      Platform.Version >= 21 &&
-      rippleColor != null
-    ) {
-      const processedColor = processColor(rippleColor);
-      invariant(
-        processedColor == null || typeof processedColor === 'number',
-        'Unexpected color given for Ripple color',
-      );
+    const {color, borderless, radius} = rippleConfig;
+    const processedColor = processColor(color);
+    invariant(
+      processedColor == null || typeof processedColor === 'number',
+      'Unexpected color given for Ripple color',
+    );
 
-      return {
-        viewProps: {
-          // Consider supporting `nativeForegroundAndroid` and `borderless`.
-          nativeBackgroundAndroid: {
-            type: 'RippleAndroid',
-            color: processedColor,
-            borderless: false,
-          },
-        },
-        onPressIn(event: PressEvent): void {
-          const view = viewRef.current;
-          if (view != null) {
-            Commands.setPressed(view, true);
-            Commands.hotspotUpdate(
-              view,
-              event.nativeEvent.locationX ?? 0,
-              event.nativeEvent.locationY ?? 0,
-            );
-          }
+    return {
+      viewProps: {
+        // Consider supporting `nativeForegroundAndroid`
+        nativeBackgroundAndroid: {
+          type: 'RippleAndroid',
+          color: processedColor,
+          borderless: borderless === true,
+          rippleRadius: radius,
         },
-        onPressMove(event: PressEvent): void {
-          const view = viewRef.current;
-          if (view != null) {
-            Commands.hotspotUpdate(
-              view,
-              event.nativeEvent.locationX ?? 0,
-              event.nativeEvent.locationY ?? 0,
-            );
-          }
-        },
-        onPressOut(event: PressEvent): void {
-          const view = viewRef.current;
-          if (view != null) {
-            Commands.setPressed(view, false);
-          }
-        },
-      };
-    }
-    return null;
-  }, [rippleColor, viewRef]);
+      },
+      onPressIn(event: PressEvent): void {
+        const view = viewRef.current;
+        if (view != null) {
+          Commands.setPressed(view, true);
+          Commands.hotspotUpdate(
+            view,
+            event.nativeEvent.locationX ?? 0,
+            event.nativeEvent.locationY ?? 0,
+          );
+        }
+      },
+      onPressMove(event: PressEvent): void {
+        const view = viewRef.current;
+        if (view != null) {
+          Commands.hotspotUpdate(
+            view,
+            event.nativeEvent.locationX ?? 0,
+            event.nativeEvent.locationY ?? 0,
+          );
+        }
+      },
+      onPressOut(event: PressEvent): void {
+        const view = viewRef.current;
+        if (view != null) {
+          Commands.setPressed(view, false);
+        }
+      },
+    };
+  }, [
```

Comment:
```
React Hook useMemo has a missing dependency: 'rippleConfig'. Either include it or remove the dependency array. (react-hooks/exhaustive-deps)
Lint code: ESLINT
Lint name: react-hooks/exhaustive-deps
```

I think you need to destructure outside of the method, and use those variables directly in the deps and in the function.